cmd/zpool/zpool_main.c

```diff
@@ -3963,11 +4069,89 @@ fsleep(float sec)
 	nanosleep(&req, NULL);
 }
 
+/* Print a "zpool status/iostat -c" script's help (-h) output. */
+static void
+print_zpool_script_help(char *name, char *path)
```

Comment:
Let's make sure `vdev_run_cmd()` and `print_zpool_script_help` use the same shared function for fork/exec.  Ideally, and updated/modified more generic version of the existing `libzfs_run_process()` would be nice.